solutions/instances/variables.tf

```diff
@@ -241,6 +246,11 @@ variable "cloud_monitoring_provision" {
   description = "Whether to create an IBM cloud monitoring instance. Set to `false` if a CRN is specified in `existing_cloud_monitoring_crn`."
   type        = bool
   default     = true
+
+  validation {
+    condition     = !(var.cloud_monitoring_provision && var.existing_cloud_monitoring_crn != null)
```

Comment:
This validation logic is not failing when `cloud_monitoring_provision = false` and `existing_cloud_monitoring_crn = null`